### PR TITLE
Update example service-account.yml for container group in documentation

### PR DESCRIPTION
**Pull request by mcen1**
##### SUMMARY

We recently stood up a container group and the pod-manager role seems insufficient as was written in the original code. This one seemed to give us enough to make AWX launch a pod in a kubernetes cluster, attach to the pod, and follow the logs. Also, updated the spec from v1beta to v1.

##### ISSUE TYPE

 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME

 - Docs

##### AWX VERSION

```
AWX 21.8.0
```

##### ADDITIONAL INFORMATION

Just updating documentation based on what I've experienced on our AWX instance in kubernetes when creating a service account in a different cluster.

```
n/a
```

### CODE REVIEW

**Comment by TheRealHaoLiu** [comment]
```
---
apiVersion: v1
kind: ServiceAccount
metadata:
  name: containergroup-service-account
  namespace: containergroup-namespace
---
kind: Role
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: role-containergroup-service-account
  namespace: containergroup-namespace
rules:
- apiGroups: [""]
  resources: ["pods"]
  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
- apiGroups: [""]
  resources: ["pods/log"]
  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
- apiGroups: [""]
  resources: ["pods/attach"]
  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
---
kind: RoleBinding
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: role-containergroup-service-account-binding
  namespace: containergroup-namespace
subjects:
- kind: ServiceAccount
  name: containergroup-service-account
  namespace: containergroup-namespace
roleRef:
  kind: Role
  name: role-containergroup-service-account
  apiGroup: rbac.authorization.k8s.io
```

from official documentation https://docs.ansible.com/automation-controller/latest/html/administration/containers_instance_groups.html#create-a-container-group

**Comment by TheRealHaoLiu** [comment]
would u mind updating your PR to contain the same information?

**Comment by TheRealHaoLiu** [comment]
@mcen1 also, welcome to the AWX community~

come hang out with us on the AWX Matrix chatroom https://matrix.to/#/#awx:ansible.com

we do have a request to make a video for AWX YouTube channel about how to setup container group in AWX 
https://github.com/ansible/awx/issues/13242#issuecomment-1329765409

If you are interested in doing that 😉 

also I been working on this demo https://github.com/TheRealHaoLiu/awx-container-group-demo to use for the video

contribution welcome <3

**Comment by mcen1** [comment]
> would u mind updating your PR to contain the same information?

Sure thing, done.

**Comment by TheRealHaoLiu** [comment]
Hi @mcen1 
```
linters: commands[1]> flake8 awx awxkit awx_collection
linters: commands[2]> yamllint -s .
./docs/container_groups/service-account.yml
  25:1      error    wrong indentation: expected 2 but found 0  (indentation)
  41:1      error    wrong indentation: expected 2 but found 0  (indentation)
  48:1      error    trailing spaces  (trailing-spaces)
```

File: docs/container_groups/service-account.yml [hunk 1]
```diff
@@ -13,30 +13,36 @@
...
+- kind: ServiceAccount
+  name: containergroup-service-account
+  namespace: containergroup-namespace
 roleRef:
-  apiGroup: rbac.authorization.k8s.io
   kind: Role
-  name: pod-manager
+  name: role-containergroup-service-account
+  apiGroup: rbac.authorization.k8s.io
+
```
Comment (TheRealHaoLiu) [inline docs/container_groups/service-account.yml]:
```suggestion

```

Comment (TheRealHaoLiu) [reply docs/container_groups/service-account.yml]: fix yamlint error 
`  48:1      error    trailing spaces  (trailing-spaces)`

File: docs/container_groups/service-account.yml [hunk 2]
```diff
@@ -13,30 +13,36 @@
...
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: awx-pod-manager
+  name: role-containergroup-service-account-binding
+  namespace: containergroup-namespace
 subjects:
-  - kind: ServiceAccount
-    name: awx
+- kind: ServiceAccount
```
Comment (TheRealHaoLiu) [inline docs/container_groups/service-account.yml:41]:
```suggestion
  - kind: ServiceAccount
```

File: docs/container_groups/service-account.yml [hunk 3]
```diff
@@ -13,30 +13,36 @@
...
+  name: containergroup-service-account
+  namespace: containergroup-namespace
```
Comment (TheRealHaoLiu) [inline docs/container_groups/service-account.yml]:
```suggestion
  - kind: ServiceAccount
    name: containergroup-service-account
    namespace: containergroup-namespace
```

Comment (TheRealHaoLiu) [reply docs/container_groups/service-account.yml]: fix yamlint 
`  41:1      error    wrong indentation: expected 2 but found 0  (indentation)`

File: docs/container_groups/service-account.yml [hunk 4]
```diff
@@ -13,30 +13,36 @@
...
-
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+- apiGroups: [""]
+  resources: ["pods/log"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+- apiGroups: [""]
+  resources: ["pods/attach"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
```
Comment (TheRealHaoLiu) [inline docs/container_groups/service-account.yml]:
```suggestion
  - apiGroups: [""]
    resources: ["pods"]
    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
  - apiGroups: [""]
    resources: ["pods/log"]
    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
  - apiGroups: [""]
    resources: ["pods/attach"]
    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
```

Comment (TheRealHaoLiu) [reply docs/container_groups/service-account.yml]: fix yamllint 
`  25:1      error    wrong indentation: expected 2 but found 0  (indentation)`

**Comment by fosterseth** [comment]
@TheRealHaoLiu the linters were addressed recently, so just needs to pass CI and final review from dev, then we can squash and merge

**Comment by mcen1** [comment]
Hello all. Is there anything needed on my side for this merge? Thanks!

**Review by TheRealHaoLiu — APPROVED** [review]

**Comment by mcen1** [comment]
Hi. Anything else needed for this to be merged?

**Pull request merged**